src/sap.ui.core/src/sap/ui/base/ManagedObject.js

```diff
@@ -3239,8 +3239,10 @@ sap.ui.define([
 	 * @param {string|function} oBindingInfo.key the name of the key property or a function getting the context as only parameter to calculate a key for entries. This can be used to improve update behaviour in models, where a key is not already available.
 	 * @param {object} [oBindingInfo.parameters] a map of parameters which is passed to the binding.
 	 * The supported parameters are listed in the corresponding model-specific implementation of <code>sap.ui.model.ListBinding</code> or <code>sap.ui.model.TreeBinding</code>.
-	 * @param {function} [oBindingInfo.groupHeaderFactory] a factory function to generate custom group visualization (optional)
-	 *
+	 * @param {function} [oBindingInfo.groupHeaderFactory] a factory function to generate custom group visualization (optional).
+	 * The factory function must return a {@link sap.m.GroupHeaderListItem}. It will be called with an <code>oGroup<code>
```

Comment:
I doubt that `sap.ui.base.ManagedObject` should define a contract that requires a `sap.m.GroupHeaderListItem`. This is true for the (standard) use case in the context of a `sap.m.List`, but not in general. 

Maybe better phrase it like

"...must return a control or element which represents the group header and which is suitable for the aggregating context (e.g. it could return an &lt;code>sap.m.GroupHeaderListItem&lt;/code> when grouping is used inside a &lt;code>sap.m.List&lt;/code> or &lt;code>sap.m.Table&lt;/code>)..."